src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -366,7 +387,7 @@ public CountingReadFilter getMergedCountingReadFilter(final SAMFileHeader samHea
             clFilters.forEach(f -> finalFilters.add(f));
         }
 
-        return aggregateFunction.apply(finalFilters, samHeader);
+        return (finalFilters.isEmpty()) ? aggregateFunction.apply(null, samHeader) : aggregateFunction.apply(finalFilters, samHeader);
```

Comment:
In addition, I added to the javadoc that the aggregateFunction should handle the empty list in the same way as `null`.